Formula/ferium.rb

```diff
@@ -0,0 +1,22 @@
+class Ferium < Formula
+  desc "Fast and multi-source CLI program for managing Minecraft mods and modpacks"
+  homepage "https://github.com/gorilla-devs/ferium"
+  url "https://github.com/gorilla-devs/ferium/archive/v4.1.7.tar.gz"
+  sha256 "2c2193417aff8af9045ea87a6ec6a35d3d1a48bdfdef05fc520fc4b9877de547"
+  license "MPL-2.0"
+  head "https://github.com/gorilla-devs/ferium.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "ferium", "help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Hmmm that might be difficult because of gorilla-devs/ferium#87

Reply:
Done 5ee8ca928afbedecd5c9266d9677d97a6f51238d

btw these tests will fail if there is not internet connection, is that fine?